packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -292,12 +292,43 @@ export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
             transform: 'translateY(-6px)'
           }
     ],
+
+    regularTick: [
+      classNames.regularTick,
+      {
+        position: 'absolute',
+        background: theme.palette.neutralTertiaryAlt,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'WindowText'
+          }
+        }
+      },
+      props.disabled && {
+        background: theme.palette.neutralLight,
+        selectors: {
+          [HighContrastSelector]: {
+            backgroundColor: 'GrayText'
+          }
+        }
+      },
+      props.vertical
```

Comment:
Instead of using a ternary in here you can assign the horizontal styles as being the default ones and only change them in case vertical is specified.